compiler/native/unmarshal.go

```diff
@@ -1,79 +0,0 @@
-// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
```

Comment:
These functions are redundant since they are identical to the ones in the [types](https://github.com/go-vela/types/blob/24f00a4a2aca371db372452f7b3b1be611883ad7/library/string.go) package.